### DISCUSSION TITLE
amp-carousel autoplay and loop aren't working

### DISCUSSION THREAD

**Discussion by purple1007** [vercel/next.js#17948] [Help] [answered]
Hi ,
I used [amp-carousel](https://playground.amp.dev/?url=https%3A%2F%2Fpreview.amp.dev%2Fdocumentation%2Fcomponents%2Famp-carousel.example.3.html%3Fformat%3Dwebsites&format=websites&_gl=1*vbs84i*_ga*YW1wLUVJbjFZVnVUamsyZl9Wem1kQWJwdEE.) and added attributes `autoplay` `loop` , but it isn't working.

```javascript
<amp-carousel
    type="slides"
    width="950"
    height="500"
    controls
    loop
    autoplay
    layout="responsive"
    delay="3000"
    role="region"
    aria-label="Looping carousel">
 ...
</amp-carousel>
```

The console print : 

```bash
error  The attribute 'autoplay' in tag 'amp-carousel' is set to the invalid value 'true'.  
https://amp.dev/documentation/components/amp-carousel/
error  The attribute 'loop' in tag 'amp-carousel' is set to the invalid value 'true'.      
https://amp.dev/documentation/components/amp-carousel/

```

Is something wrong with my code?  or how can I use correct method `amp-carousel` with loop and autoplay?

**Comment by c0b41** [comment, marked as answer]
try empty string, `controls=""`

**Reply by purple1007** [reply]
Thank you! 
It's working!!

**Reply by hichamZe** [reply]
try autoplay=""
